test/shared/controllers/ControllerBaseSpec.scala

```diff
@@ -61,11 +61,9 @@ abstract class ControllerBaseSpec
 trait ControllerTestRunner extends MockEnrolmentsAuthService with MockMtdIdLookupService with MockIdGenerator with RealAppConfig {
   _: ControllerBaseSpec =>
 
-  protected val correlationId          = "X-123"
-  protected val validNino              = "AA123456A"
-  protected val validTaxYear           = "2020"
-  protected val parsedNino: Nino       = Nino(validNino)
-  protected val parsedTaxYear: TaxYear = TaxYear(validTaxYear)
+  protected val correlationId    = "X-123"
+  protected val validNino        = "AA123456A"
+  protected val parsedNino: Nino = Nino(validNino)
```

Comment:
Had to edit this to bring it back in line with shared as edited in https://github.com/hmrc/individual-losses-api/pull/507 conflicted after rebase